File: src/main/golang/leetcode/editor/cn/[268]Missing Number.go
package lc

//leetcode submit region begin(Prohibit modification and deletion)
func missingNumber(nums []int) (res int) {
	n := len(nums)
	res ^= n
	for i, num := range nums {
		res ^= i ^ num
	}
	return
}

//leetcode submit region end(Prohibit modification and deletion)

File: src/main/golang/leetcode/editor/cn/[268]Missing Number.py
from typing import List


# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        return n * (n + 1) // 2 - sum(nums)
# leetcode submit region end(Prohibit modification and deletion)
